doublependulum.urdf

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0"?>
```

Comment:
Remove this file if it is not needed. Otherwise, put it in the `examples` folder